backend/impl/src/test/resources/setup-with-multiple-remotes.sh-discover.txt

```diff
@@ -1,21 +1,13 @@
   master  <2 files>
-  |
-  | HOTFIX Add the trigger (amended)
-  o-hotfix/add-trigger * (diverged from remote2)  <3 files>
 
   develop  <3 files>
-  |
-  | Allow ownership links
-  | 1st round of fixes
-  x-allow-ownership-link (ahead of remote1)  <4 files>
-  | |
-  | | Build arbitrarily long chains
-  | x-build-chain (untracked)  <4 files>
-  |
-  | Call web service
-  | 1st round of fixes
-  | 2nd round of fixes
-  o-call-ws (ahead of remote1)  <6 files>
-    |
-    | Drop unneeded SQL constraints
-    x-drop-constraint (untracked)  <6 files>
+
+  allow-ownership-link (ahead of remote1)  <4 files>
```

Comment:
Side effect of removing reflogs in this script: `git machete discover` yields some dumb results. Anyway, I lost interest in improving `discover` long time ago anyway, that's a futile effort anyway :trollface: